client/src/components/AddTrack.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import $ from 'jquery';
+
+class AddTrack extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      songs: []
+    };
+
+  }
+  
+
+  componentDidMount() {
+    // this.setState(this.props.onLoad);
+    
+  }
+  
+
+  
+  render() {
+    // console.log(this.props.onLoad);
+    // console.log(this.props);
+    // console.log(this.state.songs);
+    const relatedSongs = this.props.onLoad;
+    console.log(relatedSongs);
```

Comment:
delete the console log and i'm not too sure if this would work here but consider doing destructuring i.e. const {onload} = this.props